packages/shorebird_cli/lib/src/commands/release/release.dart

```diff
@@ -1,2 +1,3 @@
 export 'release_android_command.dart';
 export 'release_command.dart';
+export 'shorebird_release_ios_command.dart';
```

Comment:
`release_ios_command.dart` for consistency

Reply:
ah oops my bad